libs/status/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@hashintel/status",
```

Comment:
There is a mismatch between `libs/status` dir and `@hashintel/status`, which suggests putting this `package.json` under `/libs/@hashintel/status`. This will make sense if we want to publish the package on npm immediately. Given that the package is `private` and its version is `0.0.0-private`,  should it be moved to `/libs/@local/status`?

Reply:
[Move status package to @local/status](https://github.com/hashintel/hash/pull/1787/commits/1ac15c4ffee7f267900ac80d3d2a88beba936c04)